CHANGELOG.md

```diff
@@ -8,46 +8,47 @@
 
 - Allow support for semicolumn and tabulation in CSV files
 - Collect current distance from OR-Tools
-- Allow initial routes when clustering methods are required
+- Allow initial routes when partitions are required. Note that, it currently cannot ensure that all missions of a route will be assigned to the same route.
```

Comment:
I do not understand this. How can we consider routes for clustering and then ignore them ? Isn't it only for dicho (I guess) that it is not available yet ?

Reply:
The earlier discussion is deleted by pushes https://github.com/Mapotempo/optimizer-api/pull/99#discussion_r551867170

@braktar  I agree @fonsecadeline that we can improve the message.

We can say it is available for preprocessing/partitions and under development for split based algorithms (split_solve and dichotomous)...

Reply:
@fonsecadeline You have to tell me what can I say, as you're the one which have worked on it https://github.com/Mapotempo/optimizer-api/commit/648f73e93068045d113b3f5793a337ff58e9b013

Reply:
You can say what @senhalil said :)